docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -137,11 +137,6 @@ The release process, at a high level, involves creating a release, publishing it
 reviewing the need for either another pre-release or a final stable tag within a short period
 (normally within 3-5 business days).
 
-During the pre-release cycle a temporary branch is created, and should only receive
```

Comment:
It's a shame we have to remove this block; but at the moment we just don't do this. Hopefully we will again soon.